diff --git a/target/linux/mediatek/dts/mt7981b-tenbay-wr3000k.dts b/target/linux/mediatek/dts/mt7981b-tenbay-wr3000k.dts
--- a/target/linux/mediatek/dts/mt7981b-tenbay-wr3000k.dts
+++ b/target/linux/mediatek/dts/mt7981b-tenbay-wr3000k.dts
@@ -1,1 +1,1 @@
-                               reg = <0x580000 0x3000000>;
+                               reg = <0x580000 0x6000000>;
@@ -1,5 +1,0 @@
-                       partition@3580000 {
-                           label = "ubi1";
-                           reg = <0x3580000 0x3000000>;
-                       };
